definitions/ext-service/summary_metrics.yml

```diff
@@ -0,0 +1,18 @@
+throughput:
+  title: Throughput
+  unit: COUNT
+  query:
+    select: count(http.server.duration)
+    from: Metric
+responseTimeMs:
+  title: Response time
+  unit: SECONDS
+  query:
```

Comment:
There's a validation error coming from these summary metrics, since the `eventId` is not optional: 

```
../definitions/ext-service/summary_metrics.yml invalid
[
  {
    "keyword": "required",
    "dataPath": "['throughput'].query",
    "schemaPath": "#/definitions/query/required",
    "params": {
      "missingProperty": "eventId"
    },
    "message": "should have required property 'eventId'"
  }
] 
```